robocop/reports.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
         self.name = 'rules_by_id'
         self.message_counter = defaultdict(int)
 
-    def add_message(self, message, **kwargs):  # noqa
```

Comment:
Those kwargs were left 'just in case this method will be extended in the future' but it doesn't make sense to keep it that way.